src/device-state.coffee

```diff
@@ -300,8 +300,7 @@ module.exports = class DeviceState extends EventEmitter
 		@config.on 'change', (changedConfig) ->
 			if changedConfig.connectivityCheckEnabled?
 				network.enableConnectivityCheck(changedConfig.connectivityCheckEnabled)
-		console.log('Starting periodic check for IP addresses')
-		network.startIPAddressUpdate (addresses) =>
+		network.startIPAddressUpdate() (addresses) =>
```

Comment:
Ugh, can't wait for all of the CS to be gone.

Reply:
Yup!